MANAGEMENT.md

```diff
@@ -21,7 +21,7 @@ Then you have to [create a release](https://github.com/blog/1547-release-your-so
 
 And then upload the release to pypi:
 
-    python setup.py register sdist --formats=gztar,zip bdist_egg upload
+    python setup.py sdist && twine upload dist/*
```

Comment:
we should be uploading wheels also, but fine as is as it already improves on previous commands.